### PR TITLE
Feature - Added `transparent` as an option

### PR DESCRIPTION
**Pull request by zaaakher**
- Added `transparent` as an option in `themeColors`
- Added `transparent` to `themeColorsWithoutShades` 

Now users should be able to generate a gradient with `transparent` as an option in `to`, `from`, and/or `via`

### CODE REVIEW

**Comment by zaaakher** [comment]
Just tested it in the vercel preview link and the generated tailwind code is correct in all possible cases. ✅

The only issue I noticed is the preview boxes don't show anything when I choose `from-transparent`

**Comment by markmead** [comment]
Thanks for this @zaaakher but that issue will need to be resolved.

File: tailwind.config.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ export default {
     'pages/**/*.{vue,js}',
   ],
   safelist: [
-    { pattern: /(from|via|to|bg)-(black|white)/ },
+    { pattern: /(from|via|to|bg)-(black|white|transparent)/ },
```
Comment (markmead) [inline tailwind.config.js:12]: This was needed.

**Review by markmead — APPROVED** [review]
Looks good to me! Just needed the config updating.

**Pull request merged**